### ISSUE TITLE
Make myset timezone clearer

### ISSUE THREAD

**Issue by cwarwicker** [cwarwicker/discord-WriterBot#49]
Fix example in help command and add help info if you just run `myset timezone` without specifying a timezone.

**Comment by imaflamingo** [comment]
I tried !myset and it asked what value are you setting?  When I replied time, it responded "updated your setting timezone to time"

Next I tried !myset timezone PST which gave me the message "Using the timezone (PST), your local time right now should be: Saturday November 9, 2019 6:20 PM (UTC+0)"  And then also "updated your setting timezone to PST".  

That is still not correct since PST is UTC+8 so I tried again

!myset timezone UTC+8 which gave similar output.

I was finally able to update successfully by passing the command "!myset timezone Saturday, November 9, 2019 10:22 AM (UTC+8)"  which gave me back the response "updated your setting timezone to Saturday, November 9, 2019 10:22 AM (UTC+8)"

**Comment by imaflamingo** [comment]
would be nice if it prompted you to give a short code for your timezone like PST or CST and the bot took care of the rest, or if it prompted you to give UTC+offset formula like UTC+8 and gave an example input string

**Comment by pantywitch** [comment]
I got mine to set correctly by typing !myset timezone America/Los_Angeles.  Still a bit wordy for a "PST" analogue though.
